docs/contributing/templates/c2c-source.md

```diff
@@ -0,0 +1,103 @@
+
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs. | Yes | The Source will retry indefinitely. | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry indefinitely. | FirstPartyGenericError |
+
+### Restarting your Source
+
+<!--
+Paste this line: {@import ../../reuse/restart-c2c-source.md}
+-->
+
+## JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the [Collector Management API](/docs/api/collector-management). See [Using JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|:--|:--|:--|:--|:--|
+| config | JSON Object  | Yes | Contains the configuration parameters for the Source. |   |
+| schemaRef | JSON Object  | Yes | Set to `{"type":"MS Graph Azure AD Reporting"}`. | not modifiable |
+| sourceType | String | Yes | Set to `Universal`. | not modifiable |
+
+### Config parameters
+
+The following table shows the **config** parameters for a {{source name}}.
+
+| Parameter | Type | Required? | Default | Description | Access |
+|:--|:--|:--|:--|:--|:--|
+| `name` | String | Yes |  | Type a desired name of the Source. The name must be unique per Collector. This value is assigned to the metadata field `_source`. | modifiable |
+| `description` | String | No | null | Type a description of the Source. | modifiable |
+| `category` | String | No | null | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | modifiable |
+| `fields` | JSON Object | No |  | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field _siemForward to enable forwarding to SIEM. | modifiable |
+| `tenant_id` | String | Yes |  | Provide the Directory (tenant) ID you got after you registered (created) the Azure Application. | modifiable |
+| `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable |
+| `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
+| `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: `Directory Audit`, `Sign-in`, and `Provisioning`. For example, for both you'd use: `["Directory Audit","Signin"]` | modifiable |
+
+### Config example
```

Comment:
Can we mention the title as JSON example?

Reply:
Done